app/app.js

```diff
@@ -75,6 +75,13 @@ app.get('/', (req, res) => {
   });
 });
 
+app.get('/landing.html', (req, res) => {
+  const setup = getSetup(req);
+  res.render('../app/views/landing.html', {
+    publicationId: PUBLICATION_ID,
+    swgJsUrl: SWG_JS_URLS[setup.script],
```

Comment:
Made a comment on this as well: it's convenient for us to have it here, but confuses things a bit for someone trying to adopt an otherwise fully static page. Let's consider inlining both of these. We lose a bit on testing, but win on being explicit. Alternatively, we can add more comments to explicitly state what these values are expected to be in the HTML page itself.

Reply:
For now I inlined it.